app/src/main/java/com/github/se/orator/ui/settings/SettingsScreen.kt

```diff
@@ -165,15 +120,27 @@ fun SettingsScreen(
                     titleContentColor = MaterialTheme.colorScheme.onSurface,
                     actionIconContentColor = MaterialTheme.colorScheme.onSurfaceVariant,
                 ))
-      },
-      content = { padding ->
-        Column(
+      }) { padding ->
+        LazyColumn(
```

Comment:
Are you sure we should use a LazyColumn here? If I remember correctly we were told in the bootcamp to use those when we have a large set of data whose size is unknown.

Reply:
This column should be scrollable, hence the LazyColumn is required.